src/Stripe.net/Services/Capabilities/CapabilityListOptions.cs

```diff
@@ -0,0 +1,8 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class CapabilityListOptions : ListOptions
```

Comment:
Technically this endpoint is special and does not support Limit/pagination. But I'd rather stay consistent since this is "temporary"